content/log/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Collection
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+
+<div class="alert alert-info">
+Datadog's log management solution is is currently in private beta. If you would like to apply to it, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+
+## Overview
+{{< img src="log/index/pipeline_sketch.png" alt="Pipelines sketch" >}}
+
+## Getting started with the Agent
+
+Collecting logs is **disabled** by default in the Datadog-Agent.
+In order to start to gather your logs you need to:
+
+1. [Install the latest Datadog Agent](https://app.datadoghq.com/account/settings#agent).
+
+/////\\\\\\<br>
+/////\\\\\\<br>
+/////\\\\\\Add specific steps once we have the definitive process<br>
+/////\\\\\\<br>
+/////\\\\\\<br>
+
+## Enabling log collection from integrations
+
+To start collecting logs for an integration, you need to uncomment the log sections in your integration yaml file, and then set its parameters values according to your infrastructure.
+
+If you don't find a log section in your file you need to use the custom file configuration below instead. 
+
+<div class="alert alert-warning">
+You might not find any log section in your yaml files during the private beta, however please find below the list of integration and their yaml file that are supported:
+</div>
+
+* Apache: [apache.yaml.example](https://github.com/DataDog/integrations-core/blob/nils/Logs-integration-beta/apache/conf.yaml.example)
+* Haproxy: [haproxy.yaml.example](https://github.com/DataDog/integrations-core/blob/nils/Logs-integration-beta/haproxy/conf.yaml.example)
+* IIS: [iis.yaml.example](https://github.com/DataDog/integrations-core/blob/nils/Logs-integration-beta/iis/conf.yaml.example)
+* Java: jmx.yaml.example
+* Mongodb: [mongodb.yaml.example](https://github.com/DataDog/integrations-core/blob/nils/Logs-integration-beta/mongo/conf.yaml.example)
+* Nginx: [nginx.yaml.example](https://github.com/DataDog/integrations-core/blob/nils/Logs-integration-beta/nginx/conf.yaml.example)
+
+## Custom log collection
+
+To send custom log file that would not be part of an existing integration. Rename `custom-logs.yaml.example` in `custom-logs.yaml` and edit those parameter:
```

Comment:
> To send custom log file that would not be part of an existing integration. Rename `custom-logs.yaml.example` in `custom-logs.yaml` and edit those parameter:

"The Datadog Agent can collect logs from files or the network (TCP or UDP) and forward them to Datadog. To configure this, rename the `custom-logs.yaml.example` file to `custom-logs.yaml` and set these options:"

Reply:
👍